vignettes/slip-days.Rmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "Applying Slip Days to Your Grading Policy"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{slip-days}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Slip days are a unique grading policy, where there is an allotted budget of allowed lateness that can be applied to certain assignments. Here's an example of what that looks like in a policy file. Note that `slip_days` are their own separate section in the grading policy and do no fall under the other `categories` grading criteria. Often, slip days are computed prior to applying other grading policies because they adjust the lateness.
+```yaml
+`r xfun::file_string(system.file("extdata", "slip-days-policy-demo.yaml", package = "gradebook"))`
+
+```
+In this policy, there are 5 allotted "slip days" for the six Lab assignments. This means that students can turn in their assignments up to 2 days late across their Lab assignments with no additional lateness penalty applied. Usually, these slip days are allotted chronologically, so if a student turns in Lab 1 5 dates late, they do not have any more slip days for the rest of the labs. Similarly, if a student turned in Lab 1 a day late, they can still turn one of the other labs at most a day late with no penalty. In addition, the lateness are rounded up to the next day, so a student will use up a slip day if their assignment is turned in 1 minute late or 23 hours late.
```

Comment:
possible rewrite: "In this policy, there are 5 allotted slip days for the six Lab assignments. This means that students can turn in Lab assignments up to 2 days late with no additional lateness penalty applied. Usually, these slip days are allotted chronologically, so if a student turns in Lab 1 2 days late, they do not have any more slip days that can be used for later Lab assignments. Similarly, if a student turned in Lab 1 a day late, they can still turn one of the other labs at most a day late with no penalty. In addition, lateness is rounded up to the next day, so a student will use up a slip day if their assignment is turned in 1 minute late or 23 hours late."